src/main/java/com/powsybl/openloadflow/network/impl/LfNetworkLoaderImpl.java

```diff
@@ -428,6 +433,7 @@ private static void createBranches(List<LfBus> lfBuses, LfNetwork lfNetwork, LfT
             LfBus lfBus2 = getLfBus(branch.getTerminal2(), lfNetwork, parameters.isBreakers());
             LfBranchImpl lfBranch = LfBranchImpl.create(branch, lfNetwork, lfBus1, lfBus2, topoConfig, parameters);
             addBranch(lfNetwork, lfBranch, report);
+            addLineAreaBoundaries(branch, lfBranch, loadingContext);
```

Comment:
Rename method to addBranchAreaBoundaries

Reply:
changed in 5a29ee99ce070c764b03988fb43c2636eb909798